.vscode/launch.json

```diff
@@ -26,7 +26,8 @@
                 }
             ],
             "port": 3001,
-            "host": "localhost"
+            "host": "localhost",
+            "justMyCode": false
```

Comment:
This change is unrelated but helped debugging this as it let me look into some libraries for issues and is a minor change.

Reply:
So changing this let you step into underline library code like pandas etc? I though you can do that even now with step into over that line of code

Reply:
Yeah, the idea is whenever you're on a line that goes into a library (like Pandas) you can Step Into and it will go into that code. It's skipped over it for me without this unless I actually downloaded the library and added it to my workspace. I think I was mostly using it for the upsert library in this case.